src/code/z_lifemeter.c

```diff
@@ -1,17 +1,446 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_lifemeter/LifeMeter_Init.s")
+static s16 sHeartsPrimColors[3][3] = { { 0x00FF, 0x0046, 0x0032 }, { 0x00FF, 0x00BE, 0x0000 }, { 0x0064, 0x0064, 0x00FF } };
+static s16 sHeartsEnvColors[3][3] = { { 0x0032, 0x0028, 0x003C }, { 0x00FF, 0x0000, 0x0000 }, { 0x0000, 0x0000, 0x00FF } };
+static s16 sHeartsPrimFactors[3][3] = { { 0x0000, 0x0000, 0x0000 }, { 0x0000, 0x0078, 0xFFCE }, { 0xFF65, 0x001E, 0x00CD } };
+static s16 sHeartsEnvFactors[3][3] = { { 0x0000, 0x0000, 0x0000 }, { 0x00CD, 0xFFD8, 0xFFC4 }, { 0xFFCE, 0xFFD8, 0x00C3 } };
+static s16 sHeartsDDPrimColors[3][3] = { { 0x00FF, 0x00FF, 0x00FF }, { 0x00FF, 0x00BE, 0x0000 }, { 0x0064, 0x0064, 0x00FF } };
```

Comment:
```suggestion
static s16 sHeartsPrimColors[3][3] = { { 255, 70, 50 }, { 255, 190, 0 }, { 100, 100, 255 } };
static s16 sHeartsEnvColors[3][3] = { { 50, 40, 60 }, { 255, 0, 0}, { 0, 0, 255 } };
static s16 sHeartsPrimFactors[3][3] = { { 0, 0, 0 }, { 0, 120, -50}, { -155, 30, 205 } };
static s16 sHeartsEnvFactors[3][3] = { { 0, 0, 0 }, { 205, -40, -60 }, { -50, -40, 195 } };
static s16 sHeartsDDPrimColors[3][3] = { { 255, 255, 255 }, { 255, 190, 0 }, { 100, 100, 255 } };
```
Colours are given in decimal for this project. The factors aren't set and stone, but given their values it looks like they were also intended to be in decimal. There's a lot of hex that needs to be converted throughout the file, go ahead and do those as well